src/iexamine.cpp

```diff
@@ -1956,70 +1959,74 @@ void iexamine::fvat_empty(player &p, const tripoint &examp)
     }
 }
 
-void iexamine::fvat_full(player &p, const tripoint &examp)
+void iexamine::fvat_full( player &p, const tripoint &examp )
 {
-    bool liquid_present = false;
-    for (int i = 0; i < (int)g->m.i_at(examp).size(); i++) {
-        if (!(g->m.i_at(examp)[i].made_of(LIQUID)) || liquid_present) {
-            g->m.add_item_or_charges(examp, g->m.i_at(examp)[i]);
+    auto items_here = g->m.i_at( examp );
+    if( items_here.empty() ) {
```

Comment:
This check is redundant. The check below the loop will catch this. Why check twice for emptiness?